### DISCUSSION TITLE
Which image shoud I download for my router.

### DISCUSSION THREAD

**Discussion by billaco** [Ysurac/openmptcprouter#1791] [Q&A] [answered]
Hello, 
I am new to mptcp. I just bought an ZBT-WG1602 router with dual sim module. I don't know which image I have to download to put it  in my router. Could someone help me please ?
Thanks in advance .

Billaco

**Comment by Ysurac** [comment, marked as answer]
It's not a supported router and not yet supported by OpenWrt.
I only support router that I own and that have Linux kernel 5.4 support, I didn't find a repo for this router for now.

**Reply by billaco** [reply]
Oh ok .Thanks for your response. 
I was happy to do what I need with openmptcprouter. I hope it will be supported soon.

One more thing you said it is not yet supported by OpenWrt but  mine  is under openWRT

![Capture d’écran 2021-04-27 195744](https://user-images.githubusercontent.com/23355701/116289711-045a7400-a793-11eb-9411-a56a2a317547.png)

Thanks a lot again.
.

**Reply by Ysurac** [reply]
I mean it's not officialy supported by OpenWrt. You can't compile it yourself from official repo.
Maybe there is a custom repo but I didn't find it. OpenWrt was preinstalled ?

**Reply by billaco** [reply]
ok sorry I didn't understand. Yes it was préinstalled. I received it like this.
